rs/bitcoin/kyt/src/fetch.rs

```diff
@@ -0,0 +1,270 @@
+use crate::state::{FetchGuardError, FetchTxStatus, FetchedTx};
+use crate::types::CheckTransactionResponse;
+use crate::{blocklist_contains, GetTxError};
+use bitcoin::Address;
+use futures::future::try_join_all;
+use ic_btc_interface::Txid;
+use std::fmt::Debug;
+
+#[cfg(test)]
+mod tests;
+
+pub fn get_tx_cycle_cost(buffer_size: u32) -> u128 {
```

Comment:
nit: rename `buffer_size` to something like `max_response_bytes`?

Reply:
Renamed.